tests/lib/rules/jsx-first-prop-new-line.js

```diff
@@ -120,6 +120,10 @@ ruleTester.run('jsx-first-prop-new-line', rule, {
   invalid: [
     {
       code: '<Foo prop="one" />',
+      output: [
+        '<Foo',
+        ' prop="one" />'
```

Comment:
Hi I have been looking at this problem. To solve this, I would have to change the schema to intake the indent size. Would this be ok?

On a side note, jsx-indent-props solves this properly but would need a second run.